src/test/features/response/routes/statement-of-means/bank-accounts.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+import '../../../../routes/expectations'
+import { Paths, StatementOfMeansPaths } from 'response/paths'
+import * as idamServiceMock from '../../../../http-mocks/idam'
+import * as draftStoreServiceMock from '../../../../http-mocks/draft-store'
+import * as claimStoreServiceMock from '../../../../http-mocks/claim-store'
+import { attachDefaultHooks } from '../../../../routes/hooks'
+import { checkAuthorizationGuards } from '../checks/authorization-check'
+import { checkAlreadySubmittedGuard } from '../checks/already-submitted-check'
+import { checkCountyCourtJudgmentRequestedGuard } from '../checks/ccj-requested-check'
+import { app } from '../../../../../main/app'
+import { checkNotDefendantInCaseGuard } from '../checks/not-defendant-in-case-check'
+import { BankAccountType } from 'response/form/models/statement-of-means/bankAccountType'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = StatementOfMeansPaths.bankAccountsPage.evaluateUri(
+  { externalId: claimStoreServiceMock.sampleClaimObj.externalId }
+)
+
+describe('Defendant response: Statement of means: account-banks', () => {
+
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+
+    const method = 'get'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
+
+    context('when user authorised', () => {
+
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'cmc-private-beta', 'defendant')
+      })
+
+      checkAlreadySubmittedGuard(app, method, pagePath)
+      checkCountyCourtJudgmentRequestedGuard(app, method, pagePath)
```

Comment:
Add `checkNotDefendantInCaseGuard` to route tests, here and for POST tests.

Reply:
ok